packages/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -156,23 +146,15 @@ export class ContextSelector extends React.Component<ContextSelectorProps, { oui
             focusTrapOptions={{ clickOutsideDeactivates: true, tabbableOptions: { displayCheck: 'none' } }}
           >
             <div className={css(styles.contextSelectorMenuSearch)}>
-              <InputGroup>
-                <TextInput
-                  value={searchInputValue}
-                  type="search"
-                  placeholder={searchInputPlaceholder}
-                  onChange={onSearchInputChange}
-                  onKeyPress={this.onEnterPressed}
-                  aria-label={searchButtonAriaLabel}
-                />
-                <Button
-                  variant={ButtonVariant.control}
-                  aria-label={searchButtonAriaLabel}
-                  onClick={onSearchButtonClick}
-                >
-                  <SearchIcon aria-hidden="true" />
-                </Button>
-              </InputGroup>
+              <SearchInput
+                aria-label={searchButtonAriaLabel}
```

Comment:
```suggestion
                submitSearchButtonLabel={searchButtonAriaLabel}
```

Right now the `searchButtonAriaLabel` is being applied to the input element inside the context selector's menu. Based on the prop description, it would make more sense to pass in `searchButtonAriaLabel` to submitSearchButtonLabel